assets/static/css/data-discovery.css

```diff
@@ -241,3 +241,22 @@ input.ui-button {
 #expand-collapse.collapsed {
     left: -12px;
 }
+
+.teal-button
+{
+    background-image:none;
+    background-color:darkcyan;
+    color:white;
+}
+.teal-button:hover
```

Comment:
If you made no changes, why is there a hover, and active?

Reply:
hover was still needed, active not

Reply:
I was mistaken, and active can even be used. But the point was that the color didn't change, so you can't see that you are hovering. Which may be ok.